test/std-test.cc

```diff
@@ -300,6 +300,11 @@ TEST(std_test, exception) {
 #endif
 }
 
+TEST(std_test, type_info) {
+  EXPECT_EQ(fmt::format("{}", typeid(std::runtime_error)),
+            "std::runtime_error");
+}
```

Comment:
I think this should be wrapped in FMT_USE_RTTI check.

Reply:
Note
```
EXPECT_EQ("std::runtime_error: Test Exception", fmt::format("{:t}", ex));
EXPECT_THAT(fmt::format("{:t}", ex), StartsWith("std::system_error: "));
EXPECT_THAT(fmt::format("{:t}", ex),
                StartsWith("std::filesystem::filesystem_error: "));
```
None of these tests would succeed without `FMT_USE_RTTI`, which is not used as guard in std-test.cc.

Reply:
This looks like a bug.